EULA.md

```diff
@@ -0,0 +1,291 @@
+resources and facilitator support while enabling academic researchers
+and developers in the open source community to participate in close
+interactions between research, development, and production operations.
+Housed at the Boston University Hariri Institute for Computing and
+Computational Science & Engineering, the MOC Alliance is a unique
+collaborative effort between higher education, non-profit entities and
+industry (MOC Alliance Collaborators). For a full list of the MOC
+Alliance Collaborators, please visit*
+*[here](https://www.bu.edu/hic/centers-initiatives-labs/mocalliance/)*.
+
+**MASS OPEN CLOUD ALLIANCE (MOC ALLIANCE) END USER LICENSE AGREEMENT**
+
+This End User License Agreement (the “Agreement”) applies to all uses of
+software and services rendered by and in relation to the Mass Open Cloud
+Alliance (the “Services”). Each Service is provided and managed by the MOC
+Alliance Collaborator(s) identified here (each a “Provider”). The
+Providers may change this Agreement at any time, and we will try to let
+you know as soon as possible about any new changes. If you do not wish
+to accept the new Agreement, you must immediately stop using the Services
+and terminate this Agreement pursuant to Section 12.
+
+1. **Description of the Services**
+
+The Services are a cloud-based system through which users can upload,
+store, download, and share content from servers maintained by the
+Provider. YOUR USE OF THE SERVICES CONSTITUTES AN ACCEPTANCE OF THIS
+AGREEMENT AND THE TERMS AND CONDITIONS CONTAINED HEREIN. If you do not
+agree to all of the terms of this Agreement, you may not use the
+Services.
+
+2. **Intellectual Property**
+
+The Services allow you to upload, store, download, access, and share
+content (“User Content”). You retain ownership of any and all
+intellectual property rights you hold in your User Content.
+
+By using the Service, you give the Provider a nonexclusive, worldwide,
+royalty-free license to host and store the User Content that you upload,
+store, download, access, and share, as well as a license to perform
+those tasks necessary to ensure continued operation of the Services.
+
+The Provider retains all legal right, title, and interest in and to the
+Services, including but not limited to graphics, user interface, the
+scripts and software used to implement the Services, and any materials
+provided to you as a part of and/or in connection with the Services,
+including any and all intellectual property rights that exist therein,
+whether registered or not, and wherever in the world they may exist.
+
+3. **Code of Conduct**
+
+In exchange for your right to use the Services, you agree not to misuse
+the Services or to help anyone else to do so. In particular, you agree
+to not do any of the following with respect to the Services (unless
+otherwise authorized in writing by the Provider for a specified research
+purpose):
+
+1) probe, scan, or test the vulnerability of any system or network;
```

Comment:
This numbered list should be an actual Markdown numbered list:

```
1. probe, scan, or test the vulnerability of any system or network;
2. breach or otherwise circumvent any security or authentication measures;
.
.
.
```